src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+export interface TabType {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type TabListProps = {
+  tabs: TabType[];
+  handleTabClick: (tabItem: TabType) => void;
+  activeTab: TabType;
+};
+
+export const Tabs: React.FC<TabListProps> = ({
+  tabs,
+  handleTabClick,
+  activeTab,
+}) => {
+  return (
+    <>
+      <ul>
+        {tabs.map((tabItem) => (
+          <li
+            key={tabItem.id}
+            className={tabItem === activeTab ? 'is-active' : ''}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tabItem.id}`}
```

Comment:
Seems that the href doesn't work here. I suppose it is because of preventDefault in the onClick action
![image](https://github.com/mate-academy/react_tabs/assets/99403198/f6138029-b20e-4974-ba02-fadb9b4430de)

Reply:
Please help, it seems as only one tab is selected a time, and the preventDefault in the onclick has been replaced. But still I get error messages that only the selected tab should be highlighted?